packages/vite-plugin/package.json

```
@@ -17,7 +17,7 @@
   "dependencies": {
     "@vanilla-extract/integration": "^1.1.0"
   },
-  "devDependencies": {
+  "peerDependencies": {
     "vite": "^2.2.3"
   }
```

To fix CI commit this suggestion, then run a `yarn` locally to update the lockfile. 

```suggestion
  "devDependencies": {
    "vite": "^2.2.3"
  },
  "peerDependencies": {
    "vite": "^2.2.3"
  }
```